cross-chain/arbitrum/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "cross-chain-arbitrum",
+  "version": "0.0.1",
+  "description": "Cross chain Arbitrum",
+  "license": "MIT",
+  "files": [
+    "artifacts/",
+    "build/contracts/",
+    "contracts/",
+    "!contracts/hardhat-dependency-compiler",
+    "!**/test/",
+    "deploy/",
+    "export/",
+    "tasks/",
+    "export.json"
+  ],
+  "scripts": {
```

Comment:
Don't we need the `prepack` and `prepublishOnly` scripts just like we have for the core `solidity` package https://github.com/keep-network/tbtc-v2/blob/main/solidity/package.json#L15 ? If so, we also need the `./scripts/prepare-artifacts.sh` script. Would be good to align both as much as we can in general.

Reply:
Agree, aligned with `solidity` here [7ebc0cf](https://github.com/keep-network/tbtc-v2/pull/554/commits/7ebc0cf504fa0430e4b146bf519e2d7c48b40f77)